simulator/scheduler/plugin/plugins.go

```diff
@@ -312,84 +283,9 @@ func mergePluginSet(defaultPluginSet, customPluginSet configv1.PluginSet) config
 	return configv1.PluginSet{Enabled: enabledPlugins, Disabled: disabled}
 }
 
-// registeredPlugins returns all registered plugins.
-//
-//nolint:funlen,cyclop
-func registeredPlugins() ([]configv1.Plugin, error) {
```

Comment:
It's no longer needed since all registered plugins are now registered in multipoint. (We can just use config.RegisteredMultiPointPlugins fn